style/water.mss

```diff
@@ -7,6 +7,7 @@
 #water-areas {
   [natural = 'glacier']::natural {
     [zoom >= 5] {
+      line-clip: true;
```

Comment:
line-clip can't be used with line-dasharray without causing mismatch errors at metatile edges.